taichi/transforms/lower_ast.cpp

```diff
@@ -4,6 +4,10 @@
 
 TLANG_NAMESPACE_BEGIN
 
+namespace {
+
+using FlattenContext = Expression::FlattenContext;
```

Comment:
Is it the same as `using Expression::FlattenContext;` (avoid typing the class name twice)? Or `using xxx = ...` is a better coding style?

Reply:
Unfortunately, `using Expression::FlattenContext;` simply won't work at the **namespace scope**. Here's the error message:

```
 error: using declaration cannot refer to class member
```

 See https://en.cppreference.com/w/cpp/language/using_declaration